scripts/scil_run_commit.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Estimate the fit between a provided tractogram and DWI. Assign a weight to each
+streamline that represent how well they explain the signal.
+Default is stick-zeppelin-ball for multi-shells data.
+
+The real output from COMMIT is:
+- fit_NRMSE.nii.gz
+    fiting error (Normalized Root Mean Square Error)
+- fit_RMSE.nii.gz
+    fiting error (Root Mean Square Error)
+- results.pickle
+    Dictionary containing the experiment parameters and final weights
+- compartment_EC.nii.gz
+- compartment_IC.nii.gz
+- compartment_ISO.nii.gz
+    Each of COMMIT compartments
+
+This script can divide the input tractogram in two using a threshold to apply
+on the streamlines' weight.
```

Comment:
Typically, the threshold should be 0, keeping only streamlines that have non-zero weight and that contribute to explain the DWI signal. Streamlines with 0 weight are essentially not necessary according to COMMIT.